lib/getFormula.js

```
@@ -0,0 +1,26 @@
+var log = require('npmlog');
+var BPromise = require('bluebird');
+var request = BPromise.promisify(require('request'));
+var YAML = require('yamljs');
+var config = require('../config');
+
+module.exports = function (bioid) {
+  var url = 'https://raw.githubusercontent.com/' + config.yamlRepo.owner +
+    '/'+ config.yamlRepo.repo + '/master/members/' + bioid + '.yaml';
+
+  return request({
+    url: url,
+    method: 'GET'
+  }).spread(function (result, body) {    
+
+    var formula;
+    try {
+      formula = YAML.parse(body);
+    } catch (e) {
+      log.error('Parsing Error', {'yaml_string': body, 'exception': e});
+      formula = null;
```

You'd probably want to just throw this error rather than return null then just abort the process.